banyanmonitorgui/banyan_monitor_gui.py

```diff
@@ -86,7 +85,7 @@ def __init__(self, back_plane_ip_address,
         self.master = Tk()
 
         # set the title of the window
-        self.master.wm_title("Banyan Monitor on the " + back_plane_ip_address + "-banyanrouter")
```

Comment:
using self.back_plane_ip_address will allow the banyan base class to figure out the actual address of the local computer.